hts-sys/build.rs

```diff
@@ -93,6 +106,7 @@ fn main() {
         .generate_comments(false)
         .blacklist_function("strtold")
         .blacklist_type("max_align_t")
+        .clang_arg("-I/usr/local/musl/x86_64-linux-musl/include")
```

Comment:
Maybe make this line conditional on being in the musl target?

Reply:
Here here, I wanted to throw away the current makeshift container in favour of Alpine linux (which is MUSL-friendly by default) so that arg would not be needed at all, but then that alpine container went "sh: 1: cargo: not found" on me, need to dig a bit further to fix the alpine container now :/

Reply:
@pmarks Fixed with the new containers, although I might need to reintroduce this `.clang_arg` since CI builds do not pass without https://github.com/rust-bio/rust-htslib/pull/193/commits/b32524b84cbd6620db5fcff4d99ff64176cdd838 in place.

Thoughts?